Microcks plugin is using a non standard library for Base64

### Describe the bug

When trying to deploy the plugin using dynamic loading with Red Hat Developer Hub, the Base64 library is not found, breaking the plugin.

### Expected behavior

Use either the platform-supported handling of Base64 or use the dependency from backstage.

### Actual behavior

```backstage error an error occured while loading dynamic backend plugin '@microcks/microcks-backstage-provider' from 'file:///opt/app-root/src/dynamic-plugins-root/microcks-microcks-backstage-provider-0.0.3' Cannot find module 'js-base64'```

### How to Reproduce?

Try to use the dynamic plugin with the microcks package 0.0.3 in Red Hat Developer Sandbox

### Microcks version or git rev

0.0.3

### Install method (`docker-compose`, `helm chart`, `operator`, `docker-desktop extension`,...)

Backstage

### Additional information

_No response_

--- a/package.json
+++ b/package.json
@@ -35,16 +35,15 @@
     "@types/express": "*",
     "express": "^4.17.3",
     "express-promise-router": "^4.1.0",
-    "winston": "^3.2.1",
     "node-fetch": "^2.6.7",
-    "yn": "^4.0.0",
-    "js-base64": "^3.7.3"
+    "winston": "^3.2.1",
+    "yn": "^4.0.0"
   },
   "devDependencies": {
     "@backstage/cli": "^0.22.11",
     "@types/supertest": "^2.0.12",
-    "supertest": "^6.2.4",
-    "msw": "^0.49.0"
+    "msw": "^0.49.0",
+    "supertest": "^6.2.4"
   },
   "files": [
     "dist",

--- a/src/clients/KeycloakConnector.ts
+++ b/src/clients/KeycloakConnector.ts
@@ -14,11 +14,10 @@
  * limitations under the License.
  */
 
-import { Base64 } from 'js-base64'
 
 export function connectAndGetOAuthToken(tokenEndpoint: string, serviceAccount: string, serviceAccountCredentials: string): Promise<string> {
 
-  const credentials: string = Base64.encode(serviceAccount + ':' + serviceAccountCredentials);
+  const credentials: string = Buffer.from(serviceAccount + ':' + serviceAccountCredentials).toString('base64');
 
   return fetch(tokenEndpoint, {
     method: 'POST',

